dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/blobstore.properties

```diff
@@ -10,4 +10,8 @@ objectstorage.swift.credentials=testing
 objectstorage.swift.tempauth.username=tester
 objectstorage.swift.tempauth.tenantname=test
 objectstorage.swift.tempauth.passheadername=X-Storage-Pass
-objectstorage.swift.tempauth.userheadername=X-Storage-User
\ No newline at end of file
+objectstorage.swift.tempauth.userheadername=X-Storage-User
+
+
+blob.export.implementation=localFile
+blob.export.localFile.directory=file://var/blobExporting
```

Comment:
Is blob export really related to blobstore? It does not seem so.
Is this configuration really mandatory? Then you must upgrade the upgrade-notes. It's not possible to have sane defaults instead?

Reply:
> Is blob export really related to blobstore? It does not seem so.

It is as blobs are stored in the blobstore.

Please note that the BlobExport API explicitly require a BlobId

Hence the logical choice to put it in this configuration file.

> Is this configuration really mandatory? Then you must upgrade the upgrade-notes. It's not possible to have sane defaults instead?

We should not require this configuration.

In case these configuration entries are missing, we should supply these viable defaults.

Reply:
> It is as blobs are stored in the blobstore.

Of course of blob exports are related to blobs which are related to blobstore. But I don't see how export is related to store. For me blobstore.properties is to configure the blob storage, it has nothing to do with handling the export. If you rename it to blob.properties, it becomes consistent, but like it it's not.

Reply:
if I rename blobstore.properties to blob.properties. Then the problem is solved, right? putting to many configuration files is not user friendly IMO

Reply:
> If you rename it to blob.properties, it becomes consistent, but like it it's not.

I propose to try reading `blob.properties` then fallback to `blobstore.properties` for compatibility purposes.

Would it be OK for you?